operatorinterface.py

```diff
@@ -102,6 +102,21 @@ def getAxisBindingOfName(name: str) -> AnalogInput:
             constants.kDriveToTargetControlButtonName
         )
 
+        self.fullExtendLeftClimber = getButtonBindingOfName(
+            constants.kLeftClimberFullExtensionName
+        )
+
+        self.fullExtendRightClimber = getButtonBindingOfName(
+            constants.kRightClimberFullExtensionName
+        )
+
+        self.toggleRightClimbPiston = getButtonBindingOfName(
+            constants.kToggleRightClimbPistonName
+        )
+
+        self.toggleLeftClimbPiston = getButtonBindingOfName(
+            constants.kToggleLeftClimbPistonName
+        )
```

Comment:
My concern with making everything separate controls is that it will be hard to keep track of as a driver. The original controls are only 4 buttons, adding all this stuff would put it at I think 7 (assuming we don't have a separate retract button)